statsd-exporter/include/mappings-gen2.yml

```diff
@@ -6,20 +6,27 @@
 # in the case of users with many Airflow instances being monitored
 
 mappings:
-  # Map dot separated stats to labels
-  - match: airflow.operator_successes_(.*)
-    match_type: regex
-    name: "airflow_operator_successes"
+  - match: "airflow.*_start"
+    name: "airflow_job_start"
+    labels:
+      job_name: "$1"
+  - match: "airflow.*_end"
+    name: "airflow_job_end"
+    labels:
+      job_name: "$1"
+  - match: "airflow.*_heartbeat_failure"
+    name: "airflow_job_heartbeat_failure"
     labels:
-      # This is here to make the Astro UI work
-      # because it's expecting an operator label
-      operator: "Value"
```

Comment:
idk if this will break anything, based on the comment above?

Reply:
It needs a label named `operator`, which still exists and is now populated with the actual operator name.